llvm-project/stage2.dot

```diff
@@ -0,0 +1,46 @@
+digraph {
+  label=<<FONT COLOR="blue">stage one (built from source) </FONT>to <FONT COLOR="purple">stage two (re-built from source)</FONT>>
+  labelloc=t
+  // inputs
+  stage_zero_musl [label="musl-dev",color=red]
+  stage_zero_cmake [label="cmake",color=red]
+  stage_zero_ninja [label="ninja",color=red]
```

Comment:
I should fix the cmake/ninja relationship to better match stage1.  Probably stage2 dot file needs more work.